projects/slap_manipulation/scripts/slap_inference.py

```diff
@@ -15,27 +20,61 @@
 STRETCH_GRIPPER_MAX = 0.6
 
 
-def create_apm_input(raw_data, p_i):
+def get_proprio(raw_data, time=0.0):
```

Comment:
i still don't know if i like this name? proprio was just the term they used in per-act

Reply:
It is also a weird vector: it has a bool gripper-action, a float gripper-width and the time of current keyframe wrt entire task.

Maybe split it up into `[robot/gripper]_state` and `task_state` (just the time)?

Reply:
thats fine with me